flashinfer/jit/cubin_loader.py

```
@@ -136,18 +136,15 @@ def download_file(
-    local_path = FLASHINFER_CUBIN_DIR / safe_urljoin(checksum_path, "checksums.txt")
-    with open(local_path, "r") as f:
-        for line in f:
-            sha256, filename = line.strip().split()
-            if ".h" in filename:
-                return sha256
-    raise ValueError(f"Invalid path: checksums.txt not found in {checksum_path}")
+    for line in checksums_bytes.splitlines():
+        sha256, filename = line.strip().split()
+        if ".h" in filename:
+            return sha256
+    raise ValueError(f"Invalid checksums.txt, no flashinferMetaInfo.h found")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)